docs/userguide/storagedriver/imagesandcontainers.md

```diff
@@ -13,25 +13,159 @@ weight = -2
 # Understand images, containers, and storage drivers
 
 To use storage drivers effectively, you must understand how Docker builds and
-stores images. Then, you need an understanding of how these images are used in  containers. Finally, you'll need a short introduction to the technologies that enable both images and container operations.
+stores images. Then, you need an understanding of how these images are used by 
+containers. Finally, you'll need a short introduction to the technologies that 
+enable both images and container operations.
 
-## Images and containers rely on layers
+## Images and layers
 
-Each Docker image references a list of read-only layers that represent filesystem differences. Layers are stacked on top of each other to form a base for a container's root filesystem. The diagram below shows the Ubuntu 15.04 image comprising 4 stacked image layers.
+Each Docker image references a list of read-only layers that represent 
+filesystem differences. Layers are stacked on top of each other to form a base 
+for a container's root filesystem. The diagram below shows the Ubuntu 15.04 
```

Comment:
seems docker not officially support ubuntu 15.04. So not good to use 15.04 here anymore. Maybe 14.04